extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/cors/CORSConfig.java

```diff
@@ -31,7 +30,7 @@
      * default: returns any requested method as valid
      */
     @ConfigItem
-    public Optional<List<HttpMethod>> methods;
+    public Optional<List<String>> methods;
```

Comment:
Why this change, it doesn't look related to dealing with `*`?

Reply:
@gsmet it change was same added possibility  use * for cors methods. Can you promt me how best to do it?

Reply:
I think it's because `*` is not in the `HttpMethod` Enum

Reply:
Ah OK, totally missed it, sorry.